jason/src/peer/media/mute_state.rs

```diff
@@ -0,0 +1,212 @@
+//! [`crate::peer::PeerConnection`]s [`super::Sender`] mute state.
+
+use derive_more::From;
+
+/// All mute states in which [`super::Sender`] can be.
```

Comment:
Please, DO NOT do such referencing in docs, ever. It makes them __literally unreadable__.